handbook/sales/index.md

```diff
@@ -27,6 +27,20 @@ The Sales team represents us and our values to customers, bringing back dollars
 
 Annual Recurring Revenue (ARR) is the dollar value of contracted recurring revenue in a (normalized) one-year period.
 
+### IARR
+
+Incremental [ARR](#arr) (IARR) is the change in ARR from one period to another.
+
+### New IARR
+
+New [IARR](#iarr) is IARR from *new customers* (i.e., organizations that were **not** existing customers at the beginning of the period). See also [expansion IARR](#expansion-iarr).
+
+### Expansion IARR
+
+Expansion [IARR](#iarr) is IARR from *existing customers* (i.e., organizations that were already customers at the beginning of the period).
+
+If within a single period a new customer signs a contract which then grows in ARR before the end of the period, the total ending ARR is all considered [new IARR](#new-iarr), not expansion IARR. For example, if Acme Corp signs a $100k contract on February 3 and then the contract expands to $200k on March 5, all $200k would be considered new IARR for Q1.
```

Comment:
Random note, but we use a mix of capital and lower case characters for currency/number notations (i.e., the "k", "m", and "b"s on dollar amounts): https://sourcegraph.com/search?q=repo:%5Egithub.com/sourcegraph/about%24+%5C%24%5B0-9%5C.%5D%2B%5BkKmMbB%5D+lang:markdown+&patternType=regexp&case=yes

Over time I've noticed that you default to lower case, so I've proposed this in https://github.com/sourcegraph/about/pull/548

Reply:
@dadlerj Thanks, I updated some more mentions elsewhere but kept those that were historical.